packages/core/src/router/router.spec.ts

```diff
@@ -0,0 +1,157 @@
+import { mapTo, tap, map } from 'rxjs/operators';
+import { Effect, Middleware } from '../effects/effects.interface';
+import { findRoute, resolveRouting } from './router';
+import { HttpRequest, HttpResponse, HttpMethod } from '../http.interface';
+import { RouteMatched, Routing } from './router.interface';
+
+describe('Router', () => {
+
+  describe('#findRoute', () => {
+    test('finds route inside collection', () => {
+      // given
+      const e1$: Effect = req$ => req$.pipe(mapTo({ body: 'test' }));
+      const e2$: Effect = req$ => req$.pipe(mapTo({ body: 'test' }));
+      const e3$: Effect = req$ => req$.pipe(mapTo({ body: 'test' }));
+      const e4$: Effect = req$ => req$.pipe(mapTo({ body: 'test' }));
+
+      const routing: Routing = [
+        { regExp: /^\/?$/, methods: { GET: { effect: e1$ }, POST: { effect: e4$ } } },
+        { regExp: /^\/group\/?$/, methods: { GET: { effect: e2$ } } },
+        { regExp: /^\/group\/nested\/foo\/?$/, methods: { POST: { effect: e3$ } } },
+      ];
+
+      // when
+      const route1 = findRoute(routing, '/group/nested/foo', 'POST');
```

Comment:
I'd add case for `foo/` (with trailing slash)

Reply:
PTAL at https://github.com/marblejs/marble/pull/53/commits/4277d3623939088eef494e6a7add032431101e15